tests/unittests.c

```diff
@@ -98,7 +99,12 @@ static unsigned long __user_text test_user_task_func2(void *arg) {
 
     va = mmap(_ptr(0xfff80000), PAGE_ORDER_4K);
     printf(USTR("mmap: %lx\n"), _ul(va));
+    if (munmap(va) != 0) {
+        printf(USTR("ERROR: munmap failed\n"));
+        ud2();
+    }
 
+    va = mmap(_ptr(0xfff80000), PAGE_ORDER_4K);
```

Comment:
Can you please extend the test to add bigger and unaligned mappings as well, like doing a 2M+32k mapping at address 0xfff7fc000, which would need to be split up into 2 x 4k mappings, one 2MB mapping and yet another 2 x 4k mappings.

Reply:
Ok, good idea.